Braze-Demo.xcodeproj/project.pbxproj

```diff
@@ -1956,7 +1980,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
```

Comment:
Are we okay with this being 15+? I assume it's bc you're using the newest APIs? (If not, then 14.x+ would be good since iOS 15 is just a few months old)